source/geometry_model/chunk.cc

```diff
@@ -41,6 +41,13 @@ namespace aspect
       point1_lon(0.0)
     {}
 
+    template <int dim>
+    Chunk<dim>::ChunkGeometry::ChunkGeometry(const ChunkGeometry &other, const double para_point1_lon)
+      :
+      ChartManifold<dim,dim>(other),
+      point1_lon(para_point1_lon)
```

Comment:
Couldn't you omit the second argument to the constructor and simply write
```
template <int dim>
￼ +    Chunk<dim>::ChunkGeometry::ChunkGeometry(const ChunkGeometry &other, const double para_point1_lon)
￼ +      :
￼ +      ChartManifold<dim,dim>(other),
￼ +      point1_lon(other.para_point1_lon)
```
? This way, you don't have to pass all of those arguments by hand. Same for the other class that has even more of these arguments.

Reply:
Yes, I realized it right after I pushed the changes =( Fixed now